libtesting-navigation-util/build.gradle

```diff
@@ -18,7 +18,8 @@ android {
 }
 
 dependencies {
-    implementation(project(':libnavigation-util'))
+    // testing modules should not transitively bring in project dependencies
+    compileOnly(project(':libnavigation-util'))
```

Comment:
This resolves a problem of a breaking internal change impacting Android Auto unit tests because the old class from v2.9.3 (which AA uses) couldn't find the removed internal function (`LoggingLevel?.atLeast`) when using this test module as dependency because it was bringing in the new version of the function (`LoggingLevel?.accepts`) instead.

Example of the error:
```
java.lang.NoSuchMethodError: 'boolean com.mapbox.navigation.utils.internal.LoggingLevelUtilKt.atLeast(com.mapbox.common.LoggingLevel, com.mapbox.common.LoggingLevel)'
	at com.mapbox.androidauto.screenmanager.MapboxScreenManager.createScreen(Unknown Source)
	at com.mapbox.androidauto.screenmanager.MapboxScreenManagerTest.createScreen will push on top of an existing screen stack(MapboxScreenManagerTest.kt:192)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.mapbox.navigation.testing.MainCoroutineRule$apply$1.evaluate(MainCoroutineRule.kt:33)
	at com.mapbox.navigation.testing.LoggingFrontendTestRule$apply$1.evaluate(LoggingFrontendTestRule.kt:43)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.robolectric.RobolectricTestRunner$HelperTestRunner$1.evaluate(RobolectricTestRunner.java:591)
	at org.robolectric.internal.SandboxTestRunner$2.lambda$evaluate$0(SandboxTestRunner.java:274)
	at org.robolectric.internal.bytecode.Sandbox.lambda$runOnMainThread$0(Sandbox.java:88)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
```